node-js-basics-101/node_modules/better-assert/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Module dependencies.
+ */
+
+var AssertionError = require('assert').AssertionError
+  , callsite = require('callsite')
```

Comment:
__[jshint]__
* Comma warnings can be turned off with 'laxcomma'.
* Misleading line break before ','; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->